pbft/pbft_operation.cpp

```diff
@@ -94,7 +137,7 @@ pbft_operation::end_commit_phase()
 operation_key_t
 pbft_operation::get_operation_key()
 {
-    return std::tuple<uint64_t, uint64_t, hash_t>(this->view, this->sequence, request_hash(this->request));
+    return std::tuple<uint64_t, uint64_t, hash_t>(this->view, this->sequence, this->request_hash);
```

Comment:
you can reduce this to just:

    return {this->view, this->sequence, request_hash(this->request, this->request_hash)};